### DISCUSSION TITLE
Does URH support GFSK

### DISCUSSION THREAD

**Discussion by EdwinFairchild** [jopohl/urh#988] [Q&A] [answered]
I have to demodulate BLE signals and I am having trouble doing so. Mostly trying to figure out the proper settings to demodulate it correctly. Any insight would be welcomed.

**Comment by EdwinFairchild** [comment, marked as answer]
figured it out i think.

**Reply by Crsarmv7l** [reply]
Would be helpful if you said what you figured out. Just now need to analyze a GFSK signal
